### DISCUSSION TITLE
Welke externe bronnen willen we indexeren?

### DISCUSSION THREAD

**Discussion by rubenvdlinde** [OpenCatalogi/.github#11] [Q&A] [answered]
Onder #9 wordt er gevraagd naar het indexeren van externe bronnen, dat roept de vraag op: Welke externe bronnen willen we indexeren?

Op dit moment staan in ieder geval op de radar:

- [ ] Andere OpenCatalogi (zie ook federatieve functionaliteit)
- [ ] Joinup.eu
- [ ] publiccode <- kunnen we van GitHub aftrekken
- [ ] [componenten catalogus](https://componentencatalogus.commonground.nl/api/) Heeft een API
- [ ] [developer.overheid API's](https://developer.overheid.nl/api/apis)
- [ ] [developer.overheid ](https://developer.overheid.nl/api/repositories)
- [ ] [developer.overheid events](https://developer.overheid.nl/api/events)
- [ ] open overheid
- [ ] data overheid
- [ ] Gemma Catalogus
- [ ] CMDB

**Comment by Makkmetp** [comment]
Worden bronnen als een CMDB, contract management systeem, etc(interne systemen van organisaties) als externe bronnen gezien?

**Comment by rubenvdlinde** [comment]
Wie kiezen er voor geen vergaarbak te zijn, de locale index bestaat uit per laag

- Laag 1
- - Datacatalogus
- - ZGW
- - OOR
- Laag 2
- - developer.overheid.nl
- - Lokale API gateway
- - NLX
- Laag 3
- - componenten catalogus
- Laag 4
- - BMPN Engine | b.v. Cammunda
- - Mavin en Enterpice Studio
- Laag 5
- - CMDB
- - Software catalogus

Overal bronnen
- huidige componenten catalogus

Naast de index zijn er werk afspraken nodig. 

- Andere Open Catalogi installaties (fedderatief)
- Componenten Catalogus
- CMDB
- e.v.t Api Register

We wijzen per laag één bron aan voor indexatie, het kan zijn dat een bron voor meerdere lagen informatie bied.

Voor overige bronnen geld een BRENG princiepe aan de hand van API's zie ook  #91 

Focus ligt echt op common ground componenten / nieuwe landschap

**Comment by RonaldvCortenberghe** [comment, marked as answer]
Bovenstaande is akkoord
